src/macros.cr

```diff
@@ -34,8 +34,8 @@
 # ```
 # record Point, x : Int32 = 0, y : Int32 = 0
 #
-# Point.new      # => Point(@x=0, @y=0)
-# Point.new y: 2 # => Point(@x=0, @y=2)
+# Point.new.to_s       # => "Point(@x=0, @y=0)"
```

Comment:
For a consistency. First example of 'src/macros.cr` is written with `#to_s`.
https://github.com/crystal-lang/crystal/blob/0.20.3/src/macros.cr#L13-L16

But, I realized it's better to fix above first example by removing `#to_s` because many uses would think why it is like you.
Thought?